cleanlab/object_detection/rank.py

```diff
@@ -103,6 +106,9 @@ def get_label_quality_scores(
        swapped examples, bad location examples, and overlooked examples.
        It is important to ensure that the weights are non-negative values and that their sum equals 1.0.
 
+    overlapping_label_check:
+       A boolean that keeps a low swap score for an object that has more than 1 label associated with it.
```

Comment:
If ``True``, annotated examples with more than one label associated with them are intentionally given a low swap score.

(or some variant of this)